interface/forms/LBF/printable.php

```diff
@@ -31,6 +33,9 @@
     $patientid = 0 + $pid; // -1 means current pid
 }
 
+$patientname = getPatientName($patientid);
```

Comment:
@juggernautsei 
Really don't need to add new functions. Use this function already in api: L-45 to 58 docblock explains.

function getPatientData($pid, $given = "*, DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS") 

e.g. $use_dob = getPatientData($pid, 'DOB ')['DOB']; // short cut..
or something like that.